tests/models/test_vcard_models.py

```diff
@@ -0,0 +1,37 @@
+import pytest
+
+from husky_directory.models.vcard import VCardAddress
+
+
+class TestVCardAddress:
+    @pytest.fixture(autouse=True)
+    def initialize(self):
+        pass
+
+    @pytest.mark.parametrize(
+        "pws_value, box_number, expected",
+        [
+            # Standard address formatting
+            (
+                "123 Main St. Anytown, WA 12345-1234",
+                "351234",
+                "Box 351234;;123 Main St.;Anytown;WA;12345-1234;US;",
+            ),
+            # Has accurate zip, but city/state not parseable, so uses defaults
```

Comment:
Not so much a problem with vcards as that we're consuming unstructured data, but yeah, not really a choice, and this marginally improves on the "old" directory's behavior by at least trying to parse what it can instead of always throwing it into the street address field with only the state listed.